public/locales/en/user.json

```diff
@@ -22,10 +23,13 @@
       },
       "title": "Delete account"
     },
+    "delete_password_warning": {
```

Comment:
Oops! My bad, now it sounds like it's used for deleting your password 😅 

Last suggestion: `delete_account_password_warning`